apps/opencs/view/render/terraintexturemode.cpp

```diff
@@ -105,10 +116,18 @@ void CSVRender::TerrainTextureMode::primaryEditPressed(const WorldspaceHitResult
 
 void CSVRender::TerrainTextureMode::primarySelectPressed(const WorldspaceHitResult& hit)
 {
+    if(hit.hit == true)
```

Comment:
Why the == true? Looks redundant.

Reply:
If I recall correctly, there were some cases where hits were invalid, but I can't remember where that happened. It's possible this is unnecessary here.

Reply:
The suggestion is that you change `if(hit.hit == true)` to the much simpler `if (hit.hit)`, not that you remove the check altogether (although if it's unnecessary anyway, then obviously it should be removed).